src/player.py

```diff
@@ -1,6 +1,10 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
 class Player():
-    def __init__(self, name, current_room):
+    def __init__(self, name=Noob, current_room):
```

Comment:
Great! now you can start playing around with the player and see if you can move through the rooms